examples/react/trimmed/src/components/views/nozzle-paa.tsx

```diff
@@ -319,57 +326,58 @@ function SummaryTable({
     [safeId],
   );
 
+  const columns = useMemo(
+    () => [
+      {
+        id: 'select',
+        header: '',
+        size: 30,
+        enableSorting: false,
+        enableColumnFilter: false,
+        enableHiding: false,
+        cell: ({ row }: any) => {
+          return (
+            <div className="flex items-center justify-center">
+              <input
+                type="checkbox"
+                checked={row.getIsSelected()}
+                onChange={row.getToggleSelectedHandler()}
+                onClick={(e) => e.stopPropagation()}
+                className="cursor-pointer size-4"
+              />
+            </div>
+          );
+        },
+      } as ColumnDef<any, any>,
+      {
+        id: groupBy,
+        accessorKey: safeId,
+        header: title,
+        enableColumnFilter: false,
+      } as ColumnDef<any, any>,
+      {
+        id: 'metric',
+        accessorKey: 'metric',
+        header: metricLabel,
+        cell: (info: any) => info.getValue()?.toLocaleString(),
+        enableColumnFilter: false,
+      } as ColumnDef<any, any>,
```

Comment:
Why is the any type being used here? We already have `ColumnDef` being typed correctly in other places.  Just use that here.